MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -897,7 +897,7 @@ private void BuildStreamVideoItem(
 
             var appliedVideoConditions = options.Profile.CodecProfiles
                 .Where(i => i.Type == CodecType.Video &&
-                    i.ContainsAnyCodec(videoStream?.Codec, container) &&
+                    i.ContainsAnyCodec(videoCodec, container) &&
```

Comment:
Maybe, but with your [one-line fix](https://github.com/jellyfin/jellyfin/pull/9723/files#diff-bde118dfa6110f699ead01698c41be0bb9b4b7690a4a0adaf62ad0697ef61d69R749), I can no longer reproduce the issue.

Reply:
Oh. So the problem was only in `GetVideoTranscodeProfile`: using `transcodingProfile.VideoCodec` there (testing with its own codecs instead of the source codec) seems wrong.

And it looks like `BuildStreamVideoItem` should definitely use profile codecs to generate the correct qualifiers for attributes.

Reply:
That's it. Without this fix, qualifiers will only be applied to the transcoder in h264-to-h264 or hevc-to-hevc and similar cases.